dolphinscheduler-server/src/main/resources/logback-worker.xml

```diff
@@ -56,10 +56,6 @@
     </appender>
     <appender name="WORKERLOGFILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>${log.base}/dolphinscheduler-worker.log</file>
-        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
-            <level>INFO</level>
-        </filter>
-        <filter class="org.apache.dolphinscheduler.server.log.WorkerLogFilter"/>
```

Comment:
Why remove the ThresholdFilter configuration?
In pr #4566, you mentioned that the log in `TaskKillProcessor.process` method does not exist in the worker.log file, and the method refered [code](https://github.com/apache/incubator-dolphinscheduler/blob/dev/dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskKillProcessor.java#L85) just uses `logger.info`, the level of which is greater than or equal to `INFO` above removed code.
Maybe you should check other code?

Reply:
1.ThresholdFilter configuration and root level configuration are INFO, so remove the ThresholdFilter configuration
2.the log in TaskKillProcessor.process method does not exist in the worker.log file, because of the WorkerLogFilter

Reply:
So `WorkerLogFilter` should be removed, but `ThresholdFilter ` shouldn't?

Reply:
ThresholdFilter configuration and root level configuration are INFO, so remove the ThresholdFilter configuration

Reply:
I see it.
It's better to remove or comment all `ThresholdFilter` configuration in `logback-master.xml` and `logback-worker.xml` instead of removing one in `logback-worker.xml`.
Good job, thx

Reply:
I still have a question that is it a good way to remove `WorkerLogFilter`. Removing `WorkerLogFilter` directly makes `WorkerLogFilter` unavailable, so can we modify `WorkerLogFilter`  to achieved the goal?
```
public FilterReply decide(ILoggingEvent event) {
    if (event.getThreadName().startsWith("Worker-")){
        return FilterReply.ACCEPT;
    }

    return FilterReply.DENY;
}
```